collectors/auth0/auth0_collector.js

```diff
@@ -27,15 +27,17 @@ const tsPaths = [
 
 const HOSTNAME_REGEXP = /^[htps]*:\/\/|\/$/gi;
 
+const STREAM_NAME = 'default';
 
 class Auth0Collector extends PawsCollector {
 
     constructor(context, creds) {
-        super(context, creds, packageJson.version);
+        super(context, creds, packageJson.version,[]);
```

Comment:
`[]` is not needed because it is already defaulted in the constructor

Reply:
sure I will update this.